ui/dashboardApp/layout/signin/index.js

```diff
@@ -172,19 +173,22 @@ function TiDBSignInForm({ registry }) {
 function App({ registry }) {
   return (
     <Root>
-      <div className={styles.container}>
-        <div className={styles.dialogContainer}>
-          <div className={styles.dialog}>
-            <TiDBSignInForm registry={registry} />
+      <Router>
```

Comment:
After testing, the TopLoadingBar can't receive the `single-spa:before-routing-event` event if it isn't wrapped by the Router component.